_includes/configuration/text-color.md

```diff
@@ -34,9 +34,9 @@ By default, the number of rows and columns is dependent of the number of colors
 
 ```js
 tinymce.init({
-  selector: "textarea",  // change this value according to your HTML
-  toolbar: "forecolor backcolor",
-  color_cols: "5"
+  selector: 'textarea',  // change this value according to your HTML
+  toolbar: 'forecolor backcolor',
+  color_cols: '5'
```

Comment:
Not a new issue, but this is incorrect. It takes a number, not a string.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb

Also fixed a lot of other things here too.